src/NewsService.Models.Dto/Requests/Filters/FindNewsFilter.cs

```diff
@@ -1,20 +1,17 @@
-﻿using Microsoft.AspNetCore.Mvc;
-using System;
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
 
 namespace LT.DigitalOffice.NewsService.Models.Dto.Requests.Filters
 {
-    public class FindNewsFilter
-    {
-        [FromQuery(Name = "authorid")]
-        public Guid? AuthorId { get; set; }
+  public class FindNewsFilter
+  {
+    [FromQuery(Name = "authorid")]
```

Comment:
write name in one style, camel case or lower case

Reply:
done